ballista/rust/core/src/execution_plans/distributed_query.rs

```diff
@@ -314,6 +307,19 @@ async fn execute_query(
 
                 break Ok(futures::stream::iter(streams).flatten());
             }
+            Some(job_status::Status::Planning(_)) => {
+                if has_status_change {
+                    info!("Job {} still in initialization ...", job_id);
+                }
+                wait_future.await;
+                prev_status = status;
+            }
+            _ => {
```

Comment:
Here I changed the behavior if previously we interpret `None` as the stage is still initializing, right now this is an error, we have a separate match arm for initialized/planned query